diff --git a/RNAppleHealthKit.podspec b/RNAppleHealthKit.podspec
index abc1234..def5678 100644
--- a/RNAppleHealthKit.podspec
+++ b/RNAppleHealthKit.podspec
@@ -27,10 +27,10 @@
   s.source           = { :git => 'https://github.com/agencyenterprise/react-native-healthkit.git', :tag => s.version.to_s }

-  s.ios.deployment_target = '9.0'
+  s.ios.deployment_target = '13.4'
   s.swift_version = '4.2'

   s.source_files = 'RCTAppleHealthKit/**/*'

   s.frameworks = ['HealthKit']
-  s.dependency 'React'
+  s.dependency 'React-Core'
 end
